activation/activation.go

```diff
@@ -1003,7 +1004,7 @@ func findFullyValidHighTickAtx(
 	})
 
 	if ctx.Err() != nil {
-		return types.ATXID{}, ctx.Err()
+		return types.ATXID{}, ErrNotFound
 	}
```

Comment:
I think `ctx.Err()` is the correct response, the function was stopped because the context was canceled, not because it didn't find an ATX.